hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterServiceImpl.java

```diff
@@ -1287,6 +1292,59 @@ private void setMembersRef(Map<Address, MemberImpl> memberMap) {
     }
 
     @Override
+    public Map<ClientType, Integer> getConnectedClientStats() {
+
+        int numberOfCppClients    = 0;
+        int numberOfDotNetClients = 0;
+        int numberOfJavaClients   = 0;
+
+        Operation clientInfoOperation = new GetConnectedClientsOperation();
+        OperationService operationService1 = node.nodeEngine.getOperationService();
+        String serviceName = ClientEngineImpl.SERVICE_NAME;
+        Map<ClientType, Integer> resultMap = new HashMap<ClientType, Integer>();
+        Map<String, ClientType> clientsMap = new HashMap<String, ClientType>();
+
+        for (MemberImpl member : node.getClusterService().getMemberList()) {
+            Address target = member.getAddress();
+            InvocationBuilder invocationBuilder = operationService1.createInvocationBuilder(serviceName,
+                    clientInfoOperation, target);
+            Future<Object> future = invocationBuilder.setTryCount(1).invoke();
+            HashMap<String, ClientType> endpoints = new HashMap<String, ClientType>();
+            try {
+                endpoints = (HashMap<String, ClientType>)future.get();
+            } catch (InterruptedException e) {
+                logger.warning("Cannot get client information from : " + target.toString());
```

Comment:
so we swallow the exception if we don't get any info from some of the members. @sancar any suggestion here?